README.md

```diff
@@ -1 +1,41 @@
-# js-capstone
\ No newline at end of file
+# mobile-portfolio
+This  project is aimed at building a beautiful porfolio page.
+
+# Built With
+HTML
+CSS
+JAVASCRIPT
+
+# Video presentation
+
+[MOVIES demo](https://youtu.be/PdEa4KJuPNk)
+
+
+
+
+# live Demo
+ To see a live demo kindly visit [MOVIES](https://amklass11.github.io/Mobile-portfolio/)
+
+# Getting Started
+This is a  microverse project aimed at grasping JAVASCRIPTS wtih html
+ css and js by building the Movie app of the website. 
+
+# Authors
+Amk lass
+
+GitHub:[@amklass11](https://github.com/amklass11)
+LinkedIn:[@AmkLAss](https://www.linkedin.com/in/amk-lass-521565196/)
+# 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+# Show your support
+Give a ⭐️ if you like this project! and cheers✨
+
+ # Acknowledgments
+Microverse offered guidance on how to make the project more professional
+
+
+# 📝 License
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that all of the links presented on readme file work as expected, and that will make your project way professional, so fix this license link
![Screenshot from 2022-06-04 09-27-17](https://user-images.githubusercontent.com/56886064/171989419-1397a03b-58b7-4701-b990-4886fc4493f3.png)